spec/models/photo_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Photo, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Do as it says.

Reply:
I think this file should be removed, I've already created a test to create a photo entry in database.